core-web/libs/dotcms-scss/angular/_mixins.scss

```diff
@@ -76,7 +76,7 @@
 
 @mixin dot-field-helper-on-input {
     position: absolute;
-    top: $spacing-5;
+    top: $helper-field-icon-top-position;
```

Comment:
Are you sure we need this?

Reply:
That value is `1.8rem`. I think it is indifferent whether we have it or not as a variable, it is not being used anywhere else